src/test/java/com/google/cloud/tools/libraries/LibrariesTest.java

```diff
@@ -74,32 +75,37 @@ public void testJson() throws URISyntaxException {
 
   private static final String[] statuses = {"early access", "alpha", "beta", "GA", "deprecated"};
 
-  private static void assertApi(JsonObject api) throws URISyntaxException {
+  private static void assertApi(JsonObject api) throws IOException {
     Assert.assertFalse(api.getString("name").isEmpty());
     Assert.assertFalse(api.getString("description").isEmpty());
     String transports = api.getJsonArray("transports").getString(0);
     Assert.assertTrue(transports + " is not a recognized transport",
         "http".equals(transports) || "grpc".equals(transports));
-    new URI(api.getString("documentation"));
+    assertReachable(api.getString("documentation"));
```

Comment:
Sorry if this sounds like a dumb question, but are we testing what we're generating or what someone is giving us?

Reply:
discussed offline